src/test/javascript/spec/fakeSpec.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+describe('Math Test', function () {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["describe" is not defined.](https://www.codacy.com/app/martzrp/booktrackr/file/2628357656/issues/source?bid=3283479&fileBranchId=3317266#l3)